hieradata/common.yaml

```diff
@@ -329,6 +329,8 @@ govuk::apps::email_alert_api::email_address_override_whitelist:
   - tim.blair@digital.cabinet-office.gov.uk
   - thomas.leese@digital.cabinet-office.gov.uk
   - camille.descartes@digital.cabinet-office.gov.uk
+  - tom.whitwell@digital.cabinet-office.gov.uk
```

Comment:
It looks like these were in alphabetical order before we got here :) Maybe we should keep them that way?

Reply:
Good spot! Now amended.